### DISCUSSION TITLE
의자 테이블에 앉았을때 테이블끼리 자동 소규모 대화였으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#14] [제안하기]
게더타운에서는 기본맵에 의자-테이블을 소규모 대화영역으로 해놓아서 의자에 앉아서 대화하는데 회의실 전체에 목소리가 퍼지네요 ㅎㅎㅎ

**Comment by shlee303** [comment]
제안 감사합니다. 현재 명시적 별도의 '회의실' 영역이 대회의실 두개인데, 작은 소규모 공간들도 추가 하겠습니다!

**Comment by shlee303** [comment]
@act-lamos 테이블 영역들을 소규모 회의실로 영역 분리처리 완료했습니다 :) 좋은 의견 감사합니다~
